pkgs/development/libraries/gtk/3.x.nix

```diff
@@ -178,6 +189,10 @@ stdenv.mkDerivation rec {
 
     chmod +x ''${files[@]}
     patchShebangs ''${files[@]}
+  '' + lib.optionalString (stdenv.buildPlatform != stdenv.hostPlatform) ''
+    # Tell the post-install script to use the build platform binaries rather than the host binaries.
+    substituteInPlace meson.build \
+      --replace "join_paths(gtk_prefix, get_option('bindir'))," "'${buildPackages.gtk3}/bin',"
```

Comment:
This looks like it could easily replace more than we want. Perhaps use a patch?

Reply:
I changed it to use a patch.